src/NServiceBus.Core/Encryption/DecryptBehavior.cs

```diff
@@ -22,13 +26,54 @@ public DecryptBehavior(EncryptionMutator messageMutator)
                 await next().ConfigureAwait(false);
                 return;
             }
+
             var current = context.Message.Instance;
-            current = messageMutator.MutateIncoming(current);
```

Comment:
@andreasohlund Its not an actual message mutator. The current name was confusing, so we renamed `EncryptionMutator` to `EncryptionInspector` to name it very differently to avoid confusion.